frontend/public/components/create-yaml.tsx

```diff
@@ -61,6 +54,7 @@ export type CreateYAMLProps = {
   match: RouterMatch<{ns: string, plural: string, appName?: string}>;
   kindsInFlight: boolean;
   kindObj: K8sKind;
+  template?: string;
```

Comment:
You can pass a YAML template to this component directly (used instead of runtime `registerTemplate()` calls).